java/openjdk-java/Portfile

```
@@ -0,0 +1,307 @@
+    set tpath /Library/Java
+    description         Openjdk 8u
+    long_description    Openjdk 8u builds of Openjdk, Open-Source implementation of the Java Platform, Standard Edition, and related projects.
+    homepage            https://openjdk.java.net
+    supported_archs     x86_64
+    fetch.type          git
+    git.url             https://git.openjdk.java.net/jdk8u
+    use_xcode           yes
+    patch {
+        reinplace "s|JDK_UPDATE_VERSION=|JDK_UPDATE_VERSION=332|g" ${worksrcpath}/common/autoconf/version-numbers
+    }
+    use_configure    yes
+    configure.cmd       sh ./configure
+    configure.pre_args  --prefix=${tpath}
+    configure.args      --with-boot-jdk=/Library/Java/JavaVirtualMachines/jdk8u192-b12/Contents/Home \
+                        --with-freetype-include=${prefix}/include/freetype2 \
+                        --with-freetype-lib=${prefix}/lib \
+                        --with-conf-name=openjdk${bv}
+    build.type          gnu
+    build.target        images
+    build.post_args     JOBS=`sysctl -n hw.ncpu`
+    use_parallel_build  no
+    depends_build       port:autoconf \
+                        port:gmake
+    depends_lib         port:openjdk8-adoption-boot \
+                        port:freetype
+    set jdkn jdk${bv}.jdk
+    set bundle_dir /build/openjdk${bv}/images/j2sdk-bundle/${jdkn}
+}
+
+subport openjdk11-boot {
+    version             11.0.2
+    set build           9
+    description         openjdk 11 builds
+    long_description    ${long_description_openjdk-ga}
+    homepage            jdk.java.net
+    supported_archs     x86_64
+    master_sites        https://download.java.net/java/GA/jdk11/${build}/GPL/
+
+    checksums           rmd160  e20276958ef069ed748d881ef1eaac7dea6e121d \
+                        sha256  f365750d4be6111be8a62feda24e265d97536712bc51783162982b8ad96a70ee \
+                        size    182670822
+
+    distname            openjdk-11.0.2_osx-x64_bin
+    worksrcdir          jdk-11.0.2.jdk
+    set jdk_path ${worksrcdir}/Contents/Home
+}
+
+subport openjdk11u {
+    # https://github.com/openjdk/jdk11u/tags
+    version             11.0.15+4
+    set bv 11.0.15
+    set tpath /Library/Java
+    description         Openjdk 11 updates
+    long_description    Openjdk 11 updates of Openjdk, Open-Source implementation of the Java Platform, Standard Edition, and related projects.
+    homepage            https://openjdk.java.net
+    supported_archs     x86_64
+    fetch.type          git
+    git.url             https://git.openjdk.java.net/jdk11u
+    use_xcode           yes
+    use_configure    yes
+    configure.cmd       sh configure
+    configure.pre_args  --prefix=${tpath}
+    configure.args      --with-boot-jdk=/Library/Java/JavaVirtualMachines/jdk-11.0.2.jdk/Contents/Home \
+                        --with-conf-name=openjdk${bv}
+    build.type          gnu
+    build.target        images
+    build.post_args     JOBS=`sysctl -n hw.ncpu` \
```

Do not call `sysctl` directly.  MacPorts already provides a variable for this: `${build.jobs}`:

```suggestion
    build.post_args     JOBS=${build.jobs} \
```
```

---

I have set to `sysctl -n hw.ncpu` instead of `${build.jobs}` because parralel build is disabled so `build.job = 1` and if I enable `parralel_build`, it would pass `-j${build.jobs}` to make which isn't supported in the OpenJDK repository and will give error.

---

Though I can remove `sysctl -n hw.ncpu` though as during configure, jobs is already set based on Random-access memory